src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,89 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin onTablet {
```

Comment:
Also according to the designs all the sections are limited by max-width and have overlay background as in this picture:
<img width="515" alt="image" src="https://github.com/mate-academy/layout_dia/assets/126275137/ff4576e4-5a82-4b0c-bd1c-0b90c2c40f8f">

Try to make your page more aligned with designs, you might need to add max-width to all the sections. Make sure it all the sections behave as they should on the designs